src/ui-lib/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss

```diff
@@ -1,90 +1,92 @@
 $grav-page-heading-logo-width: 250px;
+// stylelint-disable-next-line scss/dollar-variable-pattern
+$trigger-breakpoint: gravy-breakpoint(medium);
 
 // Note: The Page Header always spans the full width of the viewport. This
 // is in case we want to give it any kind of full-bleed background styling.
 // The child elements within therefore need to respect the maximum content
 // width ($grav-page-content-max-width) themselves, where necessary.
-body > header {
-  $trigger-breakpoint: gravy-breakpoint(medium);
-
-  > div {
-    @include grav-l-container;
+body > header > div {
+  @include grav-l-container;
+
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: $grav-sp-m;
+  padding-bottom: $grav-sp-m;
+  align-items: center;
+  justify-content: space-between;
+
+  @media (min-width: $trigger-breakpoint) {
+    padding-top: 0;
+  }
+}
 
-    display: flex;
-    flex-wrap: wrap;
-    padding-top: $grav-sp-m;
-    padding-bottom: $grav-sp-m;
-    align-items: center;
-    justify-content: space-between;
+.grav-c-page-header__logo {
+  width: 70%; // 2/3 ish of the space
 
-    @media (min-width: $trigger-breakpoint) {
-      padding-top: 0;
-    }
+  a {
+    display: block;
   }
 
-  .grav-c-page-header__logo {
-    width: 70%; // 2/3 ish of the space
-
-    @media (min-width: $trigger-breakpoint) {
-      max-width: 250px;
-      margin: $grav-sp-m $grav-sp-m 0 0;
-      flex-shrink: 0;
-    }
+  @media (min-width: $trigger-breakpoint) {
+    max-width: 250px;
+    margin: $grav-sp-m $grav-sp-m 0 0;
+    flex-shrink: 0;
   }
+}
 
-  .grav-c-logo {
-    display: block;
-    max-height: 33px; // Otherwise IE11 makes it 150px tall :-(
-    fill: $grav-co-neutral-asphalt;
+.grav-c-logo {
+  display: block;
+  max-height: 33px; // Otherwise IE11 makes it 150px tall :-(
+  fill: $grav-co-neutral-asphalt;
 
-    @media (min-width: $trigger-breakpoint) {
-      width: 100%;
-    }
+  @media (min-width: $trigger-breakpoint) {
+    width: 100%;
   }
+}
 
-  /* stylelint-disable-next-line selector-max-compound-selectors */
-  a:hover .grav-c-logo {
-    fill: $grav-co-neutral-warm-grey;
-  }
+/* stylelint-disable-next-line selector-max-compound-selectors */
```

Comment:
Same thing here and in the following blocks. Please move this back within `body > header`, so that we don't break these components when they occur outside of a page header.